packages/orbit-components/src/utils/mediaQuery/README.md

```diff
@@ -55,44 +12,50 @@ You can use the following media queries in your project:
+The `TOKEN` object contains the breakpoints as keys, associated with the corresponding name of the token as value.
 
-The `getBreakpointWidth` function accepts the viewport name and a theme object. You can use it like this:
+```ts
+export const TOKEN = {
+  mediumMobile: "widthBreakpointMediumMobile",
+  largeMobile: "widthBreakpointLargeMobile",
+  tablet: "widthBreakpointTablet",
+  desktop: "widthBreakpointDesktop",
+  largeDesktop: "widthBreakpointLargeDesktop",
+} as const;
+```
 
-```jsx
-import * as React from "react";
-import { render } from "@testing-library/react";
-import theme from "@kiwicom/orbit-components/lib/defaultTheme";
-import { getBreakpointWidth } from "@kiwicom/orbit-components/lib/utils/mediaQuery";
+## getBreakpointWidth function
 
-import StyledComponent from "./";
+In case you need to access the value of the breakpoint (for testing purposes, for instance) you can use the `getBreakpointWidth` function.
 
-describe("StyledComponent", () => {
-  it("should have 100% width by default", () => {
-    render(<StyledComponent data-test="test" />);
-    expect(screen.getByTestId("test")).toHaveStyleRule("width", "100%");
-  });
+This function can also be used to return formatted strings useful for event listeners, for example.
 
-  it("should have a width of 50% on a desktop viewport", () => {
-    render(<StyledComponent data-test="test" />);
-    expect(screen.getByTestId("test")).toHaveStyleRule("width", "50%", {
-      media: getBreakpointWidth("desktop", theme),
-    });
-  });
-});
+```ts
+getBreakpointWidth(breakpoint: keyof typeof TOKEN, theme: Theme, pure?: boolean) => string | number
 ```
+
+### Arguments
+
+| Name       | Type                 | Description                                                                                                                                                                       |
+| :--------- | :------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| breakpoint | `keyof typeof TOKEN` | The name of the breakpoint.                                                                                                                                                       |
+| theme      | `Theme`              | The theme object.                                                                                                                                                                 |
+| pure       | `boolean`            | Optional. If `true`, the function will return the value as number. If false, it will return a string `"(min-width: ${VAL}px)"`, where `VAL` is the value of the breakpoint width. |
+
+To test your components with Enzyme, especially styles, you can also use the `getBreakpointWidth` function.
```

Comment:
Replace Enzyme with Jest.

Reply:
Deleted, as testing is already mentioned above. This was a leftover from the previous docs. Thanks